front/src/styles/variables.css

```diff
@@ -22,4 +22,8 @@
   --user-menu-active-color: #195292;
   --user-menu-disabled-color: #B6B6B6;
   --mmt-avatar-color: #195292;
+  --history-route-transport-icon-color: #00C853;
+  --history-route-details-color: #195292;
```

Comment:
it seems we already have color, migth we need more common name? it seems as mono pallet